### PR TITLE
cortex-m-rt fix ARM.extab

### PR DESCRIPTION
**Pull request by jacobrosenthal**
https://github.com/rust-embedded/cortex-m-rt/issues/157

### CODE REVIEW

**Comment by sajattack** [comment]
What does it do in plain english?

**Comment by jacobrosenthal** [comment]
An upstream rust change broke cortex-m-rt  a while back. Anyone fresh building will eval up to latest and not notice anything, but people who have a lock file might be stuck unless we push the minimum up above 0.6.10 I think? but I just set it to 0.6.12.

**Comment by sajattack** [comment]
Github says it needs a rebase.
